__tests__/app.test.js

```diff
@@ -47,6 +47,56 @@ describe("/api/topics", () => {
   });
 });
 
+describe.only("/api/articles/:article_id/comments", () => {
```

Comment:
Please don't leave .only in your pr code. Please remove and resubmit.

Reply:
Bugger - sorry, that was unintentional.  Will resubmit asap